torch/csrc/jit/custom_operator.h

```diff
@@ -66,12 +86,25 @@ template <
     typename... Types,
     size_t... Is>
 ReturnType callOperatorWithTuple(
+    const FunctionSchema& schema,
     Implementation&& implementation,
     Stack& stack,
     std::tuple<Types...>& tuple,
     Indices<Is...>) {
+  Node* node = nullptr;
+  if (jit::tracer::isTracing()) {
+    node = getTracedNode<Is...>(schema, tuple);
+  }
+
   pop(stack, std::get<Is>(tuple)...);
-  return std::forward<Implementation>(implementation)(std::get<Is>(tuple)...);
+  auto result =
+      std::forward<Implementation>(implementation)(std::get<Is>(tuple)...);
+
+  if (jit::tracer::isTracing()) {
+    jit::tracer::postRecordTrace(node, result);
```

Comment:
If this is what causes all the headaches with `postRecordTrace` overloads, can't we just fix it here by making it conform to the `ArrayRef<Variable>` interface?

Reply:
Our user interface exposes `at::Tensor`, so we have to handle the conversion ourselves unfortunately